### ISSUE TITLE
Generic Property entity_category not supported

### ISSUE THREAD

**Issue by SirUli** [keesschollaart81/vscode-home-assistant#1962]
Currently ``entity_category`` configurations aren't supported. Example:

```YAML
knx:
  binary_sensor:
    - name: "Some Name"
      state_address: "9/3/1"
      device_class: problem
      entity_category: diagnostic
```

This gives the following error:

```
Property entity_category is not allowed
```

While this isn't useful in the KNX integration anyway (since there is no logical grouping into devices unfortunately), it is generally not implemented in this extension. Looking at [this page](https://developers.home-assistant.io/docs/core/entity/#generic-properties) it becomes apparent that this is completely ok and could be used this way.

**Comment by SirUli** [comment]
Still relevant, sorry.

**Comment by frenck** [comment]
Fixed in #2166 and will be part of the next release

**Comment by SirUli** [comment]
Thanks Franck! Awesome!
